src/main/java/io/SchoolManagementSystem/Attendance.java

```diff
@@ -0,0 +1,9 @@
+package io.SchoolManagementSystem;
+
+public class Attendance {
```

Comment:
This design is telling the user, that we have to add records for the attendance that will have a one to one mapping with an event and a student, meaning that if we have say 10 students then we would be creating 10 records with same event object but with different student objects. If we want to filter which students were there in a specific event then It will become hard since we have to iterate the whole list of Attendance[].
I think it would be better if we group a list of students for a particular event.